aayush1607/task1/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+   <head>
+      <title>Task1-SignUpForm</title>
+   </head>
+	
+   <body  style="padding: 10px; background-color: rgb(178, 178, 250);" >
+      <h1>Create a New Account</h1>
+      <br>
+      <img src="icon.png" alt="Sorry for icon not available" width="200"
+         height="185">

```

Comment:
Write meaningful `alt` texts. This will help in SEO.
```suggestion
      <img src="icon.png" alt="facebook icon" width="200"
         height="185">
```
Also, put styles in external CSS stylesheet.

Reply:
Yes definitely i will do it.Thank You!